convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -80,11 +83,14 @@ def run(self):
         logger.info("Searching for the following excluded packages:\n")
 
         pkgs_removed = []
+        # Since the MD5 checksum of original path is used in backup path to avoid
+        # conflicts in backup folder, preparing the path is needed.
+        backedup_reposdir = os.path.join(BACKUP_DIR, hashlib.md5(DEFAULT_YUM_REPOFILE_DIR.encode()).hexdigest())
```

Comment:
@hosekadam, would you mind applying the same fix for https://github.com/r0x0d/convert2rhel/blob/host-metering/convert2rhel/pkgmanager/handlers/yum/__init__.py#L78? 

I think this is the only place left out that needs the same workaround

Reply:
@r0x0d Definitely, I missed that one.

Reply:
Just pushed a commit that fixes that: 20437fc7a59757e0c9dcba926a73958ae57023c5. 

Not the best commit, but should do the trick.